executor/executor_test.go

```diff
@@ -2464,6 +2465,66 @@ func (s *testSuite) TestMiscellaneousBuiltin(c *C) {
 	}
 }
 
+func (s *testSuite) TestSchemaCheckerSQL(c *C) {
```

Comment:
It seems not a test about any executor. Can we move this test to `session_test.go` ?